src/components/organisms/AssetContent/Pricing/FormPricing/index.tsx

```diff
@@ -9,6 +9,7 @@ import { PriceOptionsMarket } from '../../../../../@types/MetaData'
 import Button from '../../../../atoms/Button'
 import { DDO } from '@oceanprotocol/lib'
 import FormHelp from '../../../../atoms/Input/Help'
+import appConfig from '../../../../../../app.config'
```

Comment:
also works, but we try to unify access to `appConfig` through our `useSiteMetadata()` hook which is populated on build time:

```tsx
const { appConfig } = useSiteMetadata()
```

This requires to add the new keys to the GraphQL query in that hook and then they become available